events/elections/2020/README.md

```diff
@@ -31,24 +31,39 @@ previous [governance meeting video] which led to this whole process.
 
 ## Eligibility
 
-Please refer to the [Steering Committee Election Charter] for:
+Please refer to the [Steering Committee Election Charter] for [Eligibility for candidacy]
 
-- [Eligibility for candidacy]
-- [Eligibility for voting] and [Eligible voters] list (not in charter)
+Eligibility for voting in 2020 is defined as:
+
+* People who had at least 50 contributions to the Kubernetes project over
+  the past year, according to a snapshot taken 2020-08-01 of the data driving the [devstats developer activity counts dashboard][devstats-dashboard], who are also [Org Members].
+  Contributions include GitHub events like creating issues, creating pr's,
+  reviewing PR's, commenting on issues, etc. For full details see
+  [the SQL query used by devstats for developer activity counts][devstats-sql].
+
+* People who have submitted the [voting exception form] and are accepted by
+  the election committee. We *explicitly* believe the above heuristic will be
+  inaccurate and not represent the entire community. Thus we provide the form
+  for those who have contributed to the project but may not meet the above
+  criteria.  Acceptance of a form submission will be defined by a simple
+  majority vote, and the criteria used during this process will be used to
+  help refine further elections.
+
+The requirement to be an [Org Member] is new this year, added by the Steering Committee to ensure that voters are following community issues.  If you otherwise qualify to vote but have not yet applied for Org Membership, then please [request an exception][voting exception form].
```

Comment:
Can we also add a note here encouraging them to consider applying for org membership? 

https://github.com/kubernetes/community/blob/master/community-membership.md#member

Reply:
added.